env.example

```diff
@@ -0,0 +1,5 @@
+REDDIT_CLIENT_ID='<reddit-app-client-id>'
+REDDIT_SECRET_TOKEN=<reddit-app-secret-token>
+REDDIT_ACCESS_TOKEN='<run-reddit_token.py-for-value>'
+DEV_GROUPME_BOT_ID='<dev-bot-id>'
+PROD_GROUPME_BOT_ID='<prod-bot-id>'
```

Comment:
This is just an example file for what goes in the `.env` so that when you're setting everything up, you have an idea what environment variables to expect. In order for `reddit_token.py` to work, you'll need to add the `REDDIT_CLIENT_ID` and `REDDIT_SECRET_TOKEN` keys to your `.env` file